03-use-cases/01-finance/retail-banking/main.ipynb

```diff
@@ -0,0 +1,1931 @@
+{
```

Comment:
<p>Curious why this is not added as a dependency in the <code>toml</code> file?</p><p>Also, even if we go the route of not adding it to the <code>toml</code>  please keep the shell escape command commented out, and add a comment above explaining why this is added.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/824/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Spoke to @printhellohetal about this, this was due to the fact that I was running Atoti in a pip installation method rather than a poetry installation method, therefore I wasn't picking up certain dependencies that are designated in our <code>poetry.lock</code> file. This is also a higher call to standardize our dev environments (broken record on devcontainers here, sorry 🙃) so that things like this don't happen, but for now we can just take out the cell and proceed with the PR. </p>